qubes-rpc-config/README

```diff
@@ -1,13 +1,36 @@
-This is directory for qrexec service additional configuration. Configuration
-file needs to be named exactly as service. Configuration format is 'key=value'
-(without spaces around '='). Lines starting with '#' are ignored.
+This is directory for qrexec service additional configuration.
+
+Configuration syntax:
+
+* Configuration file needs to be named exactly as service.
+
+* Configuration format: key value pair that must be separated by an equal sign
+  '=' between the key and the value, while any number of spaces or tabs from
+  the key to the equal sign or from the equal sign to the value are also
+  supported, e.g. "key=value", "key   =  value", "key	=	value".
+
+* Comments: Lines starting with hashtag/octothorpe/pound sign "#" are ignored.
+
+* Boolean values: "true" and "false".
+
+* String values: must be enclosed by single quotes "'", e.g. 'str'.
 
 Supported settings:
 
-* wait-for-session - wait for full GUI session initialization before starting
-  the service. This is done using /etc/qubes-rpc/qubes.WaitForSession script.
-  There is no timeout - if the session is never initialized
-  (for example because there is no GUI running at all), service will never be
-  started.
-  Allowed values are 0 or 1.
+* force-user: Enforce that service should be run by the specified username.
+  Useful to set the user at the target qube where it can be verified that it
+  exists instead of having to modify the policy in dom0.
+  Accepted values: string
+
+* wait-for-session: Wait for full GUI session initialization before starting
+  the service. Controlled by the RPC service qubes.WaitForSession. There is
+  no timeout, if the session is never initialized (for example because there
+  is no GUI running at all), service will never be started.
+  Accepted values: true (default), false, 0 and 1 accepted for backwards
```

Comment:
The default is "false".
(I guess got this from `qvm-run`, which by default uses a service that has `wait-for-session=true` set)